app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,25 @@
 class PostsController < ApplicationController
-  def index; end
+  def index
+    @user = current_user
+    @posts = Post.where(author_id: @user.id).order(:id)
+  end
 
-  def show; end
+  def show
+    @post = Post.find(params[:id])
+    @user = User.find(@post.author_id)
+    @comments = Comment.where(post_id: @post.id)
+  end
+
+  def created
+    @post = Post.create(post_params)
+    @post.update_comments_counter
+    @post.update_likes_counter
+    redirect_back(fallback_location: root_path)
+  end
```

Comment:
- [ ] The function name should be `create` as this is what rails CRUD operation naming convention is. Please update this.

Reply:
Done ✅ .. Thanks for your review 👍🏼